tests/devapps/XamarinDev/XamarinDev/App.xaml.cs

```diff
@@ -13,9 +13,11 @@ public partial class App : Application
         public static PublicClientApplication MsalPublicClient;
 
         public static object RootViewController { get; set; }
-
-        public const string DefaultClientId = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842"; // in msidentity-samples-testing tenant -> PublicClientSample
-
+#if !IS_APPCENTER_BUILD
+        public const string DefaultClientId = "4a1aa1d5-c567-49d0-ad0b-cd957a47f842";
```

Comment:
can you leave the comment above about the client_id (4a1..842) being in msidentity-samples? so we know where to go to update things? thx.